frontend/packages/react-app/src/App.tsx

```diff
@@ -1,25 +1,52 @@
 import React, { Component, useContext, createContext } from "react";
 import { BrowserRouter as Router, Route } from "react-router-dom";
-import TopPage from "./components/pages/TopPage";
-import CreateTokenPage from "./components/pages/CreateTokenPage";
 import ExplorePage from "./components/pages/ExplorePage";
 import DashboardPage from "./components/pages/DashboardPage";
 import TokenDetailPage from "./components/pages/TokenDetailPage";
-import WithdrawAudiusPage from "./components/pages/WithdrawAudiusPage";
-import AudiusDisributionPage from "./components/pages/AudiusDisributionPage";
+import ExternalTokenDetailPage from "./components/pages/ExternalTokenDetailPage";
+import SelectDistributersPage from "./components/pages/SelectDistributersPage";
+import NewDistributerPage from "./components/pages/NewDistributerPage";
 
 const App = () => {
   return (
     <div>
       <Router>
         <div>
-          <Route exact path="/" component={TopPage} />
-          <Route path="/create" component={CreateTokenPage} />
-          <Route path="/explore" component={ExplorePage} />
-          <Route path="/dashboard" component={DashboardPage} />
-          <Route path="/token/:address" component={TokenDetailPage} />
-          <Route path="/audius-token" component={WithdrawAudiusPage} />
-          <Route path="/audius/:address" component={AudiusDisributionPage} />
+          <Route exact path="/" component={ExplorePage} />
+
+          {/* For Creator */}
+          <Route exact path="/dashboard" component={DashboardPage} />
+          <Route
+            path="/dashboard/:address"
+            component={ExternalTokenDetailPage}
+          />
+          <Route
+            path="/dashboard/distributers"
+            component={SelectDistributersPage}
+          />
+          <Route
+            exact
+            path="/dashboard/distributer/:address"
+            component={NewDistributerPage}
+          />
+          <Route
+            path="/dashboard/distributer/:address/:campaignId"
+            component={NewDistributerPage}
```

Comment:
Should component be like `DistributerDetailPage` but not `NewDistributerPage`?

Reply:
Or is this just a placeholder until implemented?

Reply:
> Should component be like `DistributerDetailPage` but not `NewDistributerPage`?

OK!